Source/Core/Core/TitleDatabase.cpp

```diff
@@ -89,6 +89,16 @@ static bool IsWiiTitle(const std::string& game_id)
   return !IsGCTitle(game_id);
 }
 
+static bool IsJapaneseGCTitle(const std::string& game_id)
+{
+  return IsGCTitle(game_id) && DiscIO::RegionSwitchGC(game_id[3]) == DiscIO::Region::NTSC_J;
+}
+
+static bool IsNonJapaneseGCTitle(const std::string& game_id)
+{
+  return IsGCTitle(game_id) && DiscIO::RegionSwitchGC(game_id[3]) != DiscIO::Region::NTSC_J;
```

Comment:
I could make a separate function that just does the Japanese part without the GC part and implement IsJapaneseGCTitle and IsNonJapaneseGCTitle in terms of that, but I don't know what to call it. It should include GC because it calls RegionSwitchGC, but I can't call it IsJapaneseGCTitle...